ledger/participant-integration-api/src/test/lib/scala/platform/indexer/ha/IndexerStabilitySpec.scala

```diff
@@ -96,21 +97,22 @@ trait IndexerStabilitySpec
         Thread.sleep(1000L)
 
         // Verify the integrity of the index database
-        if (haModeSupported) {
-          storageBackend.verifyIntegrity()(connection)
-          info(s"Integrity of the index database was checked")
-        } else {
-          val error = intercept[RuntimeException] {
-            storageBackend.verifyIntegrity()(connection)
-          }
-          assert(error.getMessage.contains("duplicate event sequential ids"))
-          info(s"Index database is corrupt, as expected")
-        }
+        storageBackend.verifyIntegrity()(connection)
+        info(s"Integrity of the index database was checked")
 
         connection.close()
         Future.successful(())
       }
       .map(_ => succeed)
+  }.transform { result =>
+    if (haModeSupported) {
+      result
+    } else {
+      // If HA mode is not supported, the test must fail, but there are multiple reasons why it can fail.
+      // E.g., duplicate parameter table initialization, or duplicate event sequential ids.
```

Comment:
oh yeah, nice we are catching this one as well :)
but here we are after the duplicate seq id-s... other type of errors might shade some real problem  :thinking: 
how about starting the indexers after each other to prevent the occurrence of the duplicate param table initialization, and assert on the duplicate sequential id errors?

Reply:
That would be possible, but then we don't test that parallel initialization does work on Postgres/Oracle, which is IMO more important to test.

Reply:
can you think of a way to test both?